src/NServiceBus.Core/Unicast/Queuing/QueueNotFoundException.cs

```diff
@@ -37,6 +37,9 @@ protected QueueNotFoundException(SerializationInfo info, StreamingContext contex
         /// </summary>
         public string Queue { get; set; }
 
+        /// <inheritdoc />
+        public override string Message => string.IsNullOrEmpty(Queue) ? base.Message : $"Queue '{Queue}' not found. This queue might need to be created manually.";
```

Comment:
```suggestion
        public override string Message => string.IsNullOrEmpty(Queue) ? base.Message : $"Queue '{Queue}' not found. This queue may have to be created manually.";
```

Reply:
sounds good, will pull this in and adjust the tests.